app/models/miq_server/worker_management/monitor/settings.rb

```diff
@@ -6,18 +6,28 @@ module MiqServer::WorkerManagement::Monitor::Settings
     attr_reader :worker_monitor_settings
   end
 
+  def reload_worker_settings
+    sync_config
+    reset_queue_messages
+    notify_workers_of_config_change(Time.now.utc)
+    MiqWorkerType.worker_classes.each(&:reload_worker_settings)
+  end
+
   def sync_config
     sync_worker_monitor_settings
     sync_child_worker_settings
     $log.log_hashes(@worker_monitor_settings)
   end
 
   def sync_child_worker_settings
-    @child_worker_settings = {}
+    child_worker_settings = {}
     MiqWorkerType.worker_classes.each do |worker_class|
-      @child_worker_settings[worker_class.settings_name] = worker_class.worker_settings
+      child_worker_settings[worker_class.settings_name] = worker_class.worker_settings
+      child_worker_settings[worker_class.settings_name].merge!(get_additional_settings(worker_class.worker_settings_paths)) unless worker_class.worker_settings_paths.empty?
```

Comment:
I didn't have a way to cache settings located in `worker_settings_paths` so decided to cache them in `@child_worker_settings`